Adds new check for ASP.NET pages with request validation set to false.

cldrn opened the pull request:
I have two versions that appear to accomplish the same thing. Which one do you think is better? The simpler version? I might be missing something. I have just starting reading the documentation.

Cheers.

semmle-cla commented:
[![CLA assistant check](https://cla.semmle.com/pull/badge/not_signed)](https://cla.semmle.com/Semmle/ql?pullRequest=2199) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.semmle.com/Semmle/ql?pullRequest=2199) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.semmle.com/check/Semmle/ql?pullRequest=2199) it.</sub>

cldrn commented:
![requestdisabled-semmle-aspnet](https://user-images.githubusercontent.com/191724/67529386-9a36af80-f681-11e9-8041-df233c0f019d.PNG)

Both rules catch all occurrences on my testing project.

cldrn commented:
Closed the other issue as I removed that local copy and started doing each PR in their own branch. At this point I have deleted the original branch for that PR. I've addressed the comments in there but can we keep working on this one?

The other PR was on master branch which became a mess when I added others there by mistake. Hence, why I started from stratch with each PR in their own branch.
